scripts/workspace-conversion/workspace-to-dataset.py

```diff
@@ -331,20 +335,11 @@ def main():
 
     # Get workspace information
     workspace = get_workspace(accessToken)
-    metadata = generate_catalog_metadata(workspace)
+    unusedWorkspaceAttributes, metadata = generate_catalog_metadata(workspace)
     print("------------------------------------------------------------")
     print("JSON Result:\n", json.dumps(metadata))
     print("------------------------------------------------------------")
-    print("Demo Link:")
-    # Add demo fields, like the server would
-    metadata["id"] = f"external_id_{uuid.uuid4()}"
-    metadata["dct:modified"] = time.time()
-    metadata["roles"] = ["reader", "admin "]
-    jsonMetadata = json.dumps(metadata)
-    print(
-        f"{terraUiRoot}/#library/browser?externalWorkspace={urllib.parse.quote(jsonMetadata)}"
-    )
+    print("Unused Attributes \n", json.dumps(unusedWorkspaceAttributes))
```

Comment:
This is something that Phil and I talked about maybe being useful from the script for debugging.